reports/2023-project-director-elections-retrospective.md

```diff
@@ -0,0 +1,112 @@
+The nomination process being done per-team helped to ensure we had good coverage across the Rust project.
+During the feedback phase, we felt like we heard from a variety of people in the project.
+
+### Ensure a Diverse Pool of Qualified Candidates for Selection
+
+We had nine qualified candidates nominated.
+Along several dimensions, diversity was rather low.
+It's hard to quantify this exactly because we did not collect demographic information about the nominees.
+
+While not usually considered diversity, given the impact of employment limits, we should perhaps consider having nominees from a diverse set of employers as part of this goal.
+
+One way to improve this going forward is to emphasize the desire to have a diverse set of candidates in our call for nominees.
+For example, we could include a statement like "the Rust Project aims to have diverse leadership that represents the perspectives of all members of the Rust community. We encourage you to consider nominees that can bring new backgrounds and perspectives to Rust leadership."
+
+### Create a Psychologically Safe Environment that Values Minority Perspectives
+
+> * Create a psychologically safe environment that values minority perspectives and encourages positive feedback, encouragement, and tolerance rather than personal preference or criticism.
+
+It's not really clear how best to evaluate this, and this is something we should pay attention to going forward.
+One option is to survey those involved using something a process like is described [here][psych-safety].
+We believe it would be best to integrate this into future election projects, or even project operations as a whole, rather than restroactively sending out a survey about the elections process at this point.
+
+[psych-safety]: https://psychsafety.co.uk/measure-psychological-safety/
+
+### Ensure Participants Feel Positive About Their Interactions and the Execution of the Process
+
+There is definitely room to improve here, as several participants had a decidedly negative experience.
+
+While this won't address all of the ways in which people had a negative experience, some of this was the result of having a new process that was changed midstream several times.
+The next time around, we will be using an existing process, although there will be adjustments made.
+We expect we will be able to give participants much clearer expectations in the future.
+
+Several other negative experiences are likely rooted in other pre-existing conflicts within the Rust community, so we encourage the Leadership Council to prioritize ways to uncover and address these conflicts.
+
+## Ensure the Process Is Compliant with Rust Foundation Bylaws and Delaware Corporation Law
+
+As far as we know, we did this.
+There were some ways in which this was challenging.
+For example, the bylaws require that we do an election to select Project Directors, but the Rust project strongly prefers consent over voting.

```

Comment:
I don't think the vote really caused problems, but it did place a tricky constraint on the process. For example, it means we have to consider what happens if somehow the consensus and the vote go in different directions. That _shouldn't_ happen, but it can.

I agree though, I think we did a good job of reconciling these norms and requirements.